note/src/main/java/com/ns/note/note/service/NoteService.java

```diff
@@ -0,0 +1,70 @@
+package com.ns.note.note.service;
+
+import com.ns.note.exception.ExceptionStatus;
+import com.ns.note.exception.ServiceException;
+import com.ns.note.note.entity.NoteEntity;
+import com.ns.note.note.repository.NoteRepository;
+import com.ns.note.note.vo.NoteRequestVo;
+import com.ns.note.note.vo.NoteResponseVo;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class NoteService {
+
+    private final NoteRepository noteRepository;
+
+    public NoteResponseVo createNewNote(NoteRequestVo vo) {
```

Comment:
```suggestion
    public NoteResponseVo createNewNote(NoteRequestVo vo) {
```

<br> 

우선 서비스와 도메인의 핵심 로직을 검증하기 위한 유닛테스트를 작성하면 좋을 것 같습니다.  

79d9ebdacd9307f61d0591df47ee64660104ada5 - 관련 e2e 테스트 작성에 관해서는 나중에 이슈 남기겠습니다.

Reply:
기획이 어느정도 안정되지 않은 상태에서 통합이나 e2e 테스트는 과하다고 생각하지만 간단한 유닛 테스트 코드 작성은 좋은 습관인것 같네요. 확인했습니다.